newIDE/app/src/Utils/GDevelopServices/Game.js

```diff
@@ -12,7 +12,8 @@ export type PublicGame = {
   authorName: string, // this corresponds to the publisher name
   publicWebBuildId?: ?string,
   description?: string,
-  authors: Array<?UserPublicProfile>,
```

Comment:
@ClementPasteau @D8H Please verify that the backend is never sending null/undefined for authors.
I'm pretty sure I had the issue and I had to fix it by adding this `?` and making the UI robust against this.